stock_available_to_promise_release_carrier_alternative/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+{
+    "name": "Stock Available to Promise Release - Carrier Alternative",
+    "summary": "Advanced selection of preferred shipping methods",
+    "version": "18.0.1.0.0",
+    "category": "Operations/Inventory/Delivery",
+    "website": "https://github.com/OCA/stock-logistics-reservation",
+    "author": "Camptocamp, BCIM, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "delivery_carrier_picking_valid",
+        "stock_available_to_promise_release",
+        "delivery_procurement_group_carrier",
+        "delivery_total_weight_from_packaging",
```

Comment:
this seems to be used only for the tests but I don't see any product packaging created. I guess this can be dropped and in the test, just use the standard weight field instead of calling `_get_estimated_weight()`

Reply:
dependency on `delivery_total_weight_from_packaging` removed